### ISSUE TITLE
`labelframes` is now supported in pprof

### ISSUE THREAD

**Issue by aalexand** [felixge/pprofutils#9]
`pprof` now has `-tagroot` and `-tagleaf` flags so the `labelframes` functionality in pprofutils is become redundant.

**Comment by felixge** [comment]
Yes. I haven't tried those new flags yet. But yeah, they might make labelframes redundant indeed, in which case I can mark it as deprecated here.

Do they allow to specify the name of the label, or do they create a virtual frame for every label?

**Comment by aalexand** [comment]
Yes, a label name can be specified. Or multiple names, comma-separated - to add a frame level for each of the specified tags. Internally at Google we have profiles with many labels, adding all labels always would make the display crowded.

Tag values are formatted according to the tag unit. Tag name is put into the filename field. If the tag value is missing for a sample, the frame is still added, but with the "unknown" value - this is done to make the number of added frame levels stable and exactly equal to the number of specified tag names in the command line flag.

**Comment by felixge** [comment]
Thanks. That's awesome. Deprecated my util, see: https://github.com/felixge/pprofutils/commit/4ab5689918f23a12d358a09e89bd206b4e1dcb26

**Comment by aalexand** [comment]
Thanks!
